src/libprojectM/Renderer/BeatDetect.hpp

```diff
@@ -32,65 +32,63 @@
 #define _BEAT_DETECT_H
 
 #include "../PCM.hpp"
-#include <algorithm>
-#include <cmath>
-
-
-// this is the size of the buffer used to determine avg levels of the input audio
-// the actual time represented in the history depends on FPS
-#define BEAT_HISTORY_LENGTH 80
+#include <array>
 
 class BeatDetect
 {
-	public:
-        // Does this really belong here? maybe belongs on projectM.Settings?
-        float beatSensitivity;
+public:
+    // Does this really belong here? maybe belongs on projectM.Settings?
```

Comment:
To answer this old question: sure, it's better to have a local copy of the value to improve speed.